### ISSUE TITLE
Troubleshooting guide

### ISSUE THREAD

**Issue by flanakin** [microsoft/finops-toolkit#734]
## ⚠️ Problem
The current troubleshooting guidance is hard to find, hard to follow, and missing a few scenarios.

## 🛠️ Solution
Create a more thorough troubleshooting guide that walks through more issues and how to debug them. Add more links to troubleshooting specific to each solution and consider ways to integrate troubleshooting into the setup process.

```[tasklist]
### Tasks
- [ ] Make the troubleshooting guide more structured
- [ ] Add the troubleshooting guide to each solution doc
- [ ] Automate hubs troubleshooting steps in the setup steps
- [ ] Automate hubs troubleshooting steps in the Data ingestion report
- [ ] Find a way to make the data dictionary more discoverable (e.g.; in the report Get started page) -- See #691
```

## 🙋‍♀️ Ask for the community

We could use your help:
1. Please vote this issue up (👍) to prioritize it.
2. Are there any issues you'd like to see covered as part of this work?

**Comment by arthurclares** [comment]
New troubleshooting guide has been published. 
https://microsoft.github.io/finops-toolkit/help/troubleshoot
